### DISCUSSION TITLE
Telegram Group - Join Hundreds of People interested in ClarionOS

### DISCUSSION THREAD

**Discussion by bytemaster** [bytemaster/clarion#15] [General]
Link:  https://t.me/clarionos

**Comment by kollab** [comment]
All are welcomed to join.
But, please keep comments / feedback positive and constructive.
Where possible, please keep the discussion focused on ClarionOS topics.
( Specifically, if you have general questions about EOS(IO), there are other forums to ask questions on these subjects )
Thanks! 😃
